OrbitQt/orbittreeview.cpp

```diff
@@ -260,6 +260,19 @@ void OrbitTreeView::mousePressEvent(QMouseEvent* event) {
   }
 }
 
+void OrbitTreeView::mouseDoubleClickEvent(QMouseEvent* event) {
+  if (model_ == nullptr) {
+    return;
+  }
+
+  QModelIndex index = indexAt(event->pos());
+  if (index.isValid()) {
+    model_->GetDataView()->OnDoubleClick(index.row());
+  } else {
+    QTreeView::mouseDoubleClickEvent(event);
+  }
+}
```

Comment:
That seems to work, but is probably not the intended way. There is a signal called `doubleClicked` which you can use. Then you don't have to deal with events. More details here: https://doc.qt.io/qt-5/qabstractitemview.html#doubleClicked

Reply:
Changed to use `doubleClicked`. Thanks for the comments!